docs/admin/kube-apiserver.md

```diff
@@ -109,6 +109,7 @@ kube-apiserver
       --service-node-port-range=: A port range to reserve for services with NodePort visibility.  Example: '30000-32767'.  Inclusive at both ends of the range.
       --ssh-keyfile="": If non-empty, use secure SSH proxy to the nodes, using this user keyfile
       --ssh-user="": If non-empty, use secure SSH proxy to the nodes, using this user name
+      --storage-media-type="application/json": The media type to use to store objects in storage. Defaults to application/json. Some resources may override this value.
```

Comment:
Sorry, the description I meant was "third party resources will never use
this".  "Some resource types may ignore this setting." would be a more
accurate message.

On Mon, Apr 25, 2016 at 7:25 PM, Daniel Smith <notifications@github.com>
wrote:

> In docs/admin/kube-apiserver.md
> <https://github.com/kubernetes/kubernetes/pull/24710#discussion_r61007216>
> :
>
> > @@ -109,6 +109,7 @@ kube-apiserver
> >        --service-node-port-range=: A port range to reserve for services with NodePort visibility.  Example: '30000-32767'.  Inclusive at both ends of the range.
> >        --ssh-keyfile="": If non-empty, use secure SSH proxy to the nodes, using this user keyfile
> >        --ssh-user="": If non-empty, use secure SSH proxy to the nodes, using this user name
> > +      --storage-media-type="application/json": The media type to use to store objects in storage. Defaults to application/json. Some resources may override this value.
>
> Hm, do we want this to maybe be a default with a separate override list?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24710/files/e4a1f3c4b34ba5ef0c8f08009e202d0b048498ff#r61007216>
>